password.py

```diff
@@ -44,12 +44,16 @@ def run(num):
   chosen = random_words(words, num)
   entropy = entropy_per_word(words)
   total_entropy = entropy*num
+  password = ""
 
   print "chosen words:"
   for word in chosen:
     print "  " + word
+    password = password+word
+
   print "entropy per word: " + str(entropy)
   print "total entropy: " + str(total_entropy)
+  print "password:\t%s" % password
```

Comment:
Could you change the tab to a space so it's consistent with the other two print statements?